src/lib/button/_button-base.scss

```diff
@@ -96,7 +94,6 @@ $mat-mini-fab-padding: 8px !default;
   border-radius: $mat-fab-border-radius;
   width: $size;
   height: $size;
-  padding: 0;
```

Comment:
Not sure about this change. On my demo the text in the fab/mini-fab are off. 
![image](https://user-images.githubusercontent.com/20677343/31908332-4cb28f44-b7eb-11e7-85f3-246f0984c2be.png)

Reply:
Must have been cache, fixed.